### PR TITLE
Patroni support documentation v2

### PR DESCRIPTION
**Pull request by pgstef**
## What Changed?

This is based on #2884 before all the restructuration.

This PR adds documentation and examples about using a supported Patroni topology.

As discussed and agreed with @martinmarques, ToDo list:
- [x] Import wording changes from the other PR
- [x] Update RHEL 8 quick-start to use EPAS and PGE 
- [x] Update Debian quick-start to Debian 11 on the same structure as RHEL 8

## Checklist

**Content**

- [X] This PR adds new content
- [ ] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

File: advocacy_docs/supported-open-source/patroni/rhel8_quick_start.mdx [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+  safety_margin: 5
+
+tags:
+  nofailover: false
+  noloadbalance: false
+  clonefrom: false
+  nosync: false
+EOF
+```
+
```
Comment (martinmarques) [inline advocacy_docs/supported-open-source/patroni/rhel8_quick_start.mdx:165]: Maybe add a few lines explaining the `replication` and `rewind` authentication section. Or the whole authentication section so the customer knows that those users with the passwords specified need to be present in the cluster

File: advocacy_docs/supported-open-source/patroni/rhel8_quick_start.mdx [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+Configure the firewall if needed to access the ports for HAProxy to listen on:
+
+```bash
+$ sudo firewall-cmd --quiet --zone=public --add-port=7000/tcp --permanent
+$ sudo firewall-cmd --quiet --zone=public --add-port=5000/tcp --permanent
+$ sudo firewall-cmd --quiet --zone=public --add-port=5001/tcp --permanent
+$ sudo firewall-cmd --quiet --reload
+```
+
+You can now use port `5000` and port `5001` to choose between read-write or read-only connection:
```
Comment (martinmarques) [inline advocacy_docs/supported-open-source/patroni/rhel8_quick_start.mdx:320]: I'd use "connections" in plural

File: advocacy_docs/supported-open-source/patroni/debian11_quick_start.mdx [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+  safety_margin: 5
+
+tags:
+  nofailover: false
+  noloadbalance: false
+  clonefrom: false
+  nosync: false
+EOF
+```
+
```
Comment (martinmarques) [inline advocacy_docs/supported-open-source/patroni/debian11_quick_start.mdx:163]: As mentioned in a comment on the RHEL quickstart, we should add a sentence mentioning how the authentication section works

File: advocacy_docs/supported-open-source/patroni/rhel8_quick_start.mdx (hunk 2 above)
Comment (pgstef) [reply advocacy_docs/supported-open-source/patroni/rhel8_quick_start.mdx]: fixed in #4b6a21d

File: advocacy_docs/supported-open-source/patroni/debian11_quick_start.mdx (hunk 3 above)
Comment (pgstef) [reply advocacy_docs/supported-open-source/patroni/debian11_quick_start.mdx:163]: See #4b6a21d for a note suggestion

**Review by martinmarques — DISMISSED** [review]

**Comment by martinmarques** [comment]
@drothery-edb I've approved this PR. Please go over with @kelpoole when to merge and push to the production website

**Pull request merged**